.github/workflows/ecosystem-compat.yml

```diff
@@ -57,3 +57,54 @@ jobs:
           cd cocotb-coverage
           export SIM=icarus
           make -k -C tests
+
+  pyuvm:
+      name: Test pyuvm 3.0.0
+      runs-on: ubuntu-latest
+      steps:
+        - name: Set up Python
+          uses: actions/setup-python@v5
+          with:
+            python-version: "3.11"
+
+        - name: Install GHDL
+          run: sudo apt-get install -y --no-install-recommends ghdl
+
+        - name: Install Icarus Verilog
+          run: sudo apt-get install -y --no-install-recommends iverilog
+
+        - name: Checkout cocotb repository
+          uses: actions/checkout@v4
+          with:
+            path: cocotb
+
+        - name: Install the development version of cocotb
+          run: pip3 install ./cocotb
+
+        - name: Checkout pyuvm repository
+          uses: actions/checkout@v4
+          with:
+            repository: pyuvm/pyuvm
+
+        - name: Install the release version of cocotb-coverage
+          run: pip3 install pyuvm==3.0.0
```

Comment:
Do we want to only test against 3.0.0? Maybe using `pyuvm~=3.0` makes sense so if patch or minor releases occur we automatically use them, but not 4.0. Or maybe it should be `pyuvm>=3`?

Reply:
I did not know ~= existed.

Though, I think >=3 makes sense.

I'll make the change (which will restart the tests.)

Reply:
Changed to >=3 and repushed.